template_package/include/template_package/template_tactical_plugin_node.hpp

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (C) <SUB><year> LEIDOS.
```

Comment:
Does it make sense to make the copyright holder a parameter as well? Just thinking in case external users end up using these templates too. Though, it's a pretty easy find/replace after the fact.

Reply:
I have found multi-line substitutions across multiple files which include * and / extremely hard to get working reliably, so I would prefer to leave the standard default. external users could easily find and replace.